test/handlers/crash_h.erl

```diff
@@ -13,4 +13,10 @@ init(_, no_reply) ->
 init(Req, reply) ->
 	_ = cowboy_req:reply(200, Req),
 	ct_helper:ignore(?MODULE, init, 2),
-	error(crash).
+	error(crash);
+init(_, internal_exit) ->
+	ct_helper:ignore(?MODULE, init, 2),
+	exit(internal_exit);
+init(_, external_exit) ->
+	ct_helper:ignore(?MODULE, init, 2),
+	exit(self(), external_exit).
```

Comment:
The ignore function is about crashes not exits. This handler is about crashes as well. I don't think this belongs here. A better place for this test would be in stream_handler_SUITE similar to this test https://github.com/ninenines/cowboy/blob/master/test/stream_handler_SUITE.erl#L341-L362 (just get the pid and exit the process immediately, the test shouldn't be that long). It's OK to only test `exit/2` in this case.